index.html

```diff
@@ -0,0 +1,191 @@
+                <div class="card-description">
+                    <h3>Tonic</h3>
+                    <div class="profile-description">
+                        <h4>CANOPY</h4>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>Back End Dev</p>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>2015</p>
+                    </div>
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="profile"><img src="image/Snapshoot Portfolio (3).png" alt="Snapshoot"></div>
+                <div class="card-description">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <div class="profile-description">
+                        <h4>CANOPY</h4>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>Back End Dev</p>
+                        <img src="image/Counter.png" alt="Counter">
+                        <p>2015</p>
+                    </div>
+                    <div class="paragraph-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="languages">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+        </section>
+
+        <section class="About-myself">
+            <h2>About <br> Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <nav class="nav-links">
+                <ul>
+                    <li><a href="#"><img src="image/github.svg" alt="twitter"></a></li>
+                    <li><a href="#"><img src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="image/Vector-1.svg" alt="Vector-1"></a></li>
+                    <li><a href="#"><img src="image/twitter.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="image/Vector.svg" alt="Vector"></a></li>
```

Comment:
[OPTIONAL]
- [ ] It would be great to link these links to your social handlers.